content/id/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -0,0 +1,245 @@
+    ```
+    atau tambahkan _field_ `staticPodPath: <direktori>` pada [berkas konfigurasi kubelet](/docs/tasks/administer-cluster/kubelet-config-file).
+
+4. Jalankan ulang kubelet. Pada Fedora, kamu dapat menjalankan:
+
+    ```shell
+    # Jalankan perintah berikut pada Node tempat kubelet berjalan
+    systemctl restart kubelet
+    ```
+
+### Manifes Pod statis pada Web {#konfigurasi-via-http}
+
+Kubelet akan secara berkala mengunduh berkas yang ditentukan pada argumen `--manifest-utl=<URL>`
+dan menginterpretasikan sebagai sebuah berkas JSON/YAML yang berisikan definisi Pod.
+Mirip dengan cara kerja [manifes pada filesystem](##konfigurasi-via-filesystem),
+kubelet akan mengambil manifes berdasarkan jadwal. Jika ada perubahan pada daftar
+Pod statis, maka kubelet akan menerapkannya.
+
+Untuk menggunakan cara ini:
+
+1. Buat sebuah berkas YAML dan simpan pada suatu web server sehingga kamu pada memberikan URL tersebut pada kubelet.
+
+    ```yaml
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: static-web
+      labels:
+        role: myrole
+    spec:
+      containers:
+        - name: web
+          image: nginx
+          ports:
+            - name: web
+              containerPort: 80
+              protocol: TCP
+    ```
+
+2. Atur kubelet pada suatu Node untuk menggunakan manifes pada web ini dengan menajalankan menggunakan argumen `--manifest-url=<url-manifes>`. Pada Fedora, ubah pada `/etc/kubernetes/kubelet` untuk menambahkan baris ini:
+
+    ```
+    KUBELET_ARGS="--cluster-dns=10.254.0.10 --cluster-domain=kube.local --manifest-url=<url-manifes>"
+    ```
+
+3. Jalankan ulang kubelet. Pada Fedora, kamu dapat menjalankan:
+
+    ```shell
+    # Jalankan perintah ini pada Node tempat kubelet berjalan
+    systemctl restart kubelet
+    ```
+
+## Mengobservasi perilaku Pod statis
+
+Ketika kubelet berjalan, secara otomatis akan menjalankan semua Pod statis yang terdefinisi.
+Ketika kamu mendefinisikan Pod statis dan menjalankan ulang kubelet, Pod statis yang baru
+akan dijalankan.
+
+Kamu dapat melihat kontainer yang berjalan (termasuk Pod statis) dengan menjalankan (pada Node):
+```shell
+# Jalankan perintah ini pada Node tempat kubelet berjalan
+docker ps
+```
+
+Hasilnya akan seperti berikut:
+
+```
+CONTAINER ID IMAGE         COMMAND  CREATED        STATUS         PORTS     NAMES
+f6d05272b57e nginx:latest  "nginx"  8 minutes ago  Up 8 minutes             k8s_web.6f802af4_static-web-fk-node1_default_67e24ed9466ba55986d120c867395f3c_378e5f3c
+```
+
+Kamu dapat melihat Pod mirror pada API server:
+
+```shell
+kubectl get pods
+```
+```
+NAME                       READY     STATUS    RESTARTS   AGE
+static-web-my-node1        1/1       Running   0          2m
+```
+
+{{< note >}}
+Pastikan kubelet memiliki izin untuk membuat Pod mirror pada API server. Jika tidak,
+pembuatannya akan ditolak oleh API server. Lihat
+[PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/).
```

Comment:
we have indonesian page for this https://kubernetes.io/id/docs/concepts/policy/pod-security-policy/
```suggestion
[PodSecurityPolicy](/id/docs/concepts/policy/pod-security-policy/).
```